### PR TITLE
Add automatic pillar of DRBD and remove the static template.

### PR DESCRIPTION
**Pull request by nick-wang**
Remove some parameters of DRBD pillar for a minimal configuration.
Enable install cluster package in habootstrap pillar.
Compat the pillar drbd-formula >= 0.3.4

### CODE REVIEW

File: pillar_examples/README.md [hunk 1]
```diff
@@ -35,3 +36,18 @@ Choose one according to your needs.
 Finally, if instead of deploying SAP HANA and the cluster together, to only
 deploy one of them update the salt/hana_node/files/salt/top.sls file only using
 the desired component and removing/commenting the other.
+
+
+# DRBD automatic pillars
+For a preconfigured environment, you can use pillars files which are in [DRBD automatic directory](./automatic/DRBD)
```
Comment (arbulu89) [inline pillar_examples/README.md]: `s/pillars/pillar`

File: pillar_examples/README.md [hunk 2]
```diff
@@ -35,3 +36,18 @@ Choose one according to your needs.
...
+
+**Could be used for testing purpose and not for production as they have default settings.**
+
+From git top-level folder, copy files:
+
+`cp pillar_examples/automatic/DRBD/{cluster.sls,drbd.sls} salt/drbd_node/files/pillar/`
```
Comment (arbulu89) [inline pillar_examples/README.md]: Why `DRBD` in capital letters? I would use small case

File: pillar_examples/automatic/DRBD/cluster.sls [hunk 3]
```diff
@@ -1,23 +1,28 @@
 cluster:
+  install_packages: true
   name: 'drbd_cluster'
-  init: '<HOSTNAME_1>'
-  interface: 'eth0'
+  init: {{ grains['name_prefix'] }}01
+  interface: eth0
```
Comment (arbulu89) [inline pillar_examples/automatic/DRBD/cluster.sls]: I would put `eth1` here as it is the internal isolated network

Comment (nick-wang) [reply pillar_examples/automatic/DRBD/cluster.sls]: Thanks, fixed in the new commit.

File: pillar_examples/README.md (hunk 2 above)
Comment (nick-wang) [reply pillar_examples/README.md]: Change the folder to lower case in the new commit.

File: pillar_examples/README.md (hunk 1 above)
Comment (nick-wang) [reply pillar_examples/README.md]: Thanks, fixed in the new commit.

File: pillar_examples/automatic/drbd/cluster.sls [hunk 4]
```diff
@@ -1,23 +1,31 @@
...
-  init: '<HOSTNAME_1>'
-  interface: 'eth0'
+  init: {{ grains['name_prefix'] }}01
+  {% if grains['provider'] == 'libvirt' %}
+  interface: eth1
+  {% else %}
+  interface: eth0
+  {% endif %}
+  unicast: True
   join_timer: 20
```
Comment (arbulu89) [inline pillar_examples/automatic/drbd/cluster.sls]: This parameter name has been changed to `wait_for_initialization`

**Pull request merged**